### PR TITLE
try logging email success and failures

### PR DESCRIPTION
**Pull request by iaincrawf**
not sure how to find the logs yet to check but will send some on sys with this there and see

- not sure if I'll get an actual error thrown if the email service fails, anyone know? 

- Does anyone know what I'm meant to put in the list of LoggingProperty? Fields I want logged like supplier and email maybe? I just put them in the message

### CODE REVIEW

File: tests/Unit/Facade.Tests/PurchaseOrderServiceTests/ContextBase.cs [hunk 1]
```diff
@@ -45,6 +46,9 @@ public class ContextBase
 
         protected IFileReader FileReader { get; private set; }
 
+        protected ILog Logger { get; private set; }
+
+
```
Comment (RossStewart) [inline tests/Unit/Facade.Tests/PurchaseOrderServiceTests/ContextBase.cs:51]: blank

File: src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs [hunk 2]
```diff
@@ -156,13 +156,13 @@ public ProcessResult SendPdfEmail(string html, string emailAddress, int orderNum
                     pdf.Result,
                     $"LinnPurchaseOrder{orderNumber}");
 
+            order.SentByMethod = "EMAIL";
+
+            //// todo When remove mini orders: stop setting sentbymethod mini order and switch to below
```
Comment (RossStewart) [inline src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs:161]: leaving the to do in?

**Review by RossStewart — APPROVED** [review]
looks good!

File: src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs (hunk 2 above)
Comment (iaincrawf) [reply src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs:161]: yeah, can't remove mini order for a while! Unless you mean the actual todo, in which case yeah happy to leave it here just so it's super obvious, the rest are in their own methods but haven't done it here for updating 1 thing. Could but doesn't seem worth it. Although I changed the code so it doesn't actually make sense will update it haha

**Review by RossStewart — APPROVED** [review]

**Pull request merged**